person.rb

```diff
@@ -1,16 +1,17 @@
 require_relative './nameable'
+require_relative './rental'
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id, :parent_permission, :rentals
+  attr_reader :id, :rentals
 
-  def initialize(age, name = 'Unknown', parent_permission: true)
+  def initialize(age, name, parent_permission: true)
+    super()
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
     @rentals = []
-    super()
   end
```

Comment:
- [ ] You are almost there! However, every time I add a `person` the age is `unknown`:

![Screen Shot 2022-07-30 at 9 29 13 AM](https://user-images.githubusercontent.com/83717285/181918944-6d153a8d-4f07-434d-984e-6aa9cdd1a5e4.png)

I tried debugging this error, I guess the bug is in the class `Person` initializer, it would be great to fix this bug. Everything else is working like a charm!